src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -32,22 +33,21 @@ class FileList extends React.Component {
     super(props);
 
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.fileList = React.createRef();
+    this.ensureActiveItemVisible = this.ensureActiveItemVisible.bind(this);
   }
 
   componentDidUpdate() {
-    // Raven is making `componentDidUpdate` fire on each heart beat thus
-    // the uploader modal jumps when users with a small screen (like a 13" mbp)
-    // scrolls down. This will check if the user's browser is about a 13" screen
-    // or less and prevent the scrolling into view from happening.
-    if (document.documentElement.clientHeight > 792) {
-      this.ensureActiveItemVisible();
-    }
+    const throttleActiveItemVisibility = throttle(this.ensureActiveItemVisible, 5000);
+    throttleActiveItemVisibility();
+
+    // fix -- description text is reset while files are processing
+
+    ['input', 'click'].forEach(event => document.addEventListener(event,throttleActiveItemVisibility));
```

Comment:
@digitalartcrew
- why do you need to check on these events?
- if you do end up needing it you would only need to add it once, not on every update